ruby_fibonacci.rb

```diff
@@ -0,0 +1,20 @@
+def fibonacci(num)
+    if num <= 0
+        puts "Invalid number supplied."
+        return
+    end
+    num_one = 0
+    num_two = 1
+    n = 0 
+    
+    num.times do |i|
```

Comment:
Codacy found an issue: [Unused block argument - `i`. You can omit the argument if you don't care about it.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6318302)